app/views/admin/poll/questions/answers/documents.html.erb

```diff
@@ -0,0 +1,52 @@
+<%= back_link_to %>
+
+<h2><%= t("admin.questions.show.answers.documents_list") %></h2>
+
+<p><%= link_to @question.poll.name, admin_poll_path(@question.poll) %> > <%= link_to @question.title, admin_question_path(@question) %> > <%= @answer.title %></p>
```

Comment:
You can change this line to:

```html
<ul class="breadcrumbs">
  <li><%= link_to @question.poll.name, admin_poll_path(@question.poll) %></li>
  <li><%= link_to @question.title, admin_question_path(@question) %></li>
  <li><%= @answer.title %></li>
</ul>
```

This have nice prefab styles by Foundation 😉